bucket/zoxide.json

```
@@ -0,0 +1,26 @@
+{
+    "version":  "0.4.3",
+    "description": "A faster way to navigate your filesystem",
+    "homepage":  "https://github.com/ajeetdsouza/zoxide",
+    "license":  "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/ajeetdsouza/zoxide/releases/download/v0.4.3/zoxide-x86_64-pc-windows-msvc.exe#/zoxide.exe",
+            "hash": "502e858569ea2dd31b08db5e6621330f36abdc9b90186c915c102fbd2c7b0c1a"
+        }
+    },
+    "bin": "zoxide.exe",
+    "env_set": {
+        "_ZO_DATA_DIR": "$dir\\data"
+    },
+    "persist": "data",
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/ajeetdsouza/zoxide/releases/download/v$version/zoxide-x86_64-pc-windows-msvc.exe#/zoxide.exe"
+            }
+        }
+    },
+    "notes": "See https://github.com/ajeetdsouza/zoxide#powershell for zoxide autoload"
```

Notes are essential information for user. They need to be in general, information part of manifest.

When users decide what to install and investigate manifest, and they will see notes about some additional configuration/requirements are needed, they will not install it in some cases.

And if the notes contain something like: `for full working version you need to do 50 steps and sacrifice goat` after, and it is hidden in lots of lines in manifest it will be counterproductive/un-userfriendly for user.

```suggestion
    }
```